tests/test_status_codes.py

```diff
@@ -0,0 +1,28 @@
+import http
+import sys
+from typing import Tuple
+
+import pytest
+
+from falcon import status_codes
+
+
+class TestStatusCodes:
```

Comment:
I would prefer to parametrize instead of a loop, then you see clearly which status fails.

Reply:
Done.